lib/runner.ts

```diff
@@ -199,7 +199,7 @@ export class Runner extends EventEmitter {
 
     var browser_ = ProtractorBrowser.wrapDriver(
         driver, config.baseUrl, config.rootElement,
-        config.untrackOutstandingTimeouts);
+        config.untrackOutstandingTimeouts, config.ng12Hybrid);
```

Comment:
Instead of adding `ng12Hybrid` to the constructor, let's set it like below.

(I don't want to keep adding to our list of constructor params).